### PR TITLE
Azure aks install guide

### PR DESCRIPTION
**Pull request by hubvu**
- Added installation instructions for Ondat on AKS under the **Install** section.
- Removed `azure-aks.md` file from the **Platforms** section in favour of the **Install** section.

### CODE REVIEW

**Review by cvlc — APPROVED** [review]
LGTM, great work @hubvu !

**Comment by cannisondat** [comment]
LGTM, let's wait for Maria's approval and we can merge :) (will notify Richa as well)

**Review by hubvu — COMMENTED** [review]
Looks good to me @mdeneva thanks for the updates

File: docs/install/microsoft-azure-aks.md
Comment (Arau) [inline docs/install/microsoft-azure-aks.md:3]: This is the title that will be rendered in the side menu. I believe it will be very long for that column and may be break.

Comment (hubvu) [reply docs/install/microsoft-azure-aks.md:3]: Good point - should we remove "Microsoft"? I did see on the "Docker Enterprise Edition" that it auto wraps the title in the side menu if you make your window smaller. 

@mdeneva what do you think?

**Pull request merged**
